RLBotFramework/utils/item_id_parser.py

```diff
@@ -1,7 +1,7 @@
 # data in the form of: ID,FullName,AssetPackageName,AsciiLabel,PackID,Quality,UnlockMethod
 
 dump_file = input('Data dump (input) file name?->')
-data = open(dump_file, 'r', encoding='utf-8-sig')
```

Comment:
Did you get rid of this encoding param on purpose?

Reply:
Yes i did, the first time i used it it needed that( i think i downloaded the txt).
This time, i copied the text and created the txt myself, and it gave me an error because it was not encoded in utf-8-sig.